### PR TITLE
Soap can be used to clean microwaves

### PR DESCRIPTION
**Pull request by ghost**
extremely controversial chef powercreep

[tweak]
:cl:
 * tweak: Soap can be used to clean microwaves.

### CODE REVIEW

**Comment by juanototo2** [comment]
MoMMI stealth buff

File: code/game/machinery/kitchen/microwave.dm [hunk 1]
```diff
@@ -115,23 +115,24 @@
...
+				"<span class='notice'>[user] starts to clean the microwave.</span>", \
+				"<span class='notice'>You start to clean the microwave.</span>" \
+			)
+			if (do_after(user, src,20))
+				if(istype(R))
+					R.reagents.remove_reagent(CLEANER,5)
 				user.visible_message( \
-					"<span class='notice'>[user] starts to clean the microwave.</span>", \
-					"<span class='notice'>You start to clean the microwave.</span>" \
+					"<span class='notice'>[user]  has cleaned  the microwave.</span>", \
```
Comment (jknpj) [inline code/game/machinery/kitchen/microwave.dm]: too many spaces

Comment (ghost) [reply code/game/machinery/kitchen/microwave.dm]: What do you mean? The relevant section wasn't changed.

Comment (ghost) [reply code/game/machinery/kitchen/microwave.dm]: Oh I see the extra spaces now, should be fixed.

**Comment by SonixApache** [comment]
muh inter-departemental co-operations

**Pull request merged**